Dockerfile

```
@@ -0,0 +1,15 @@
+FROM chefes/releng-base
+ENV VAULT_ADDR="https://vault.es.chef.co"
+ENV VAULT_NAMESPACE="releng"
+WORKDIR /workdir
+COPY . /workdir
+RUN git clone https://github.com/chef-partners/azure-chef-extension.git
+WORKDIR /workdir/azure-chef-extension
+RUN bundle install
+RUN git config --global core.autocrlf false
+WORKDIR /workdir
+RUN chmod +x /workdir/publicenvvar.sh
+#for public delivery
+CMD "/workdir/publicenvvar.sh"
```

ideally we should not repeat the path for the WORKDIR. All commands will get executed relative to the working directory path. So could have used the following instead:

```suggestion
CMD "./publicenvvar.sh"
```